content/docs/command-reference/import-url.md

```diff
@@ -109,8 +109,12 @@ $ dvc run -n download_data \
           wget https://data.dvc.org/get-started/data.xml -O data.xml
 ```
 
-`dvc import-url` generates an import stage `.dvc` file and `dvc run` a regular
-stage (in `dvc.yaml`).
+`dvc import-url` generates an <abbr>import stage</abbr> `.dvc` file and
+`dvc run` a regular stage (in `dvc.yaml`).
+
+⚠️ DVC won't push or pull imported data to
```

Comment:
`pull to remote` sounds strange

Reply:
Oops, thanks. Updated to "to/from" in f23fc5ce.